task_processing/plugins/mesos/mesos_executor.py

```diff
@@ -14,6 +14,7 @@
     ExecutionFramework
 )
 from task_processing.plugins.mesos.translator import mesos_status_to_event
+from task_processing.plugins.smartstack import get_mesos_master
```

Comment:
So basically, you're saying we just add a smartstack module that users can call before creating an executor.